multisig2/src/lib.rs

```diff
@@ -449,17 +457,17 @@ mod tests {
     use std::fmt::{Debug, Error, Formatter};
 
     use near_sdk::test_utils::VMContextBuilder;
+    use near_sdk::Balance;
     use near_sdk::{testing_env, PublicKey};
     use near_sdk::{AccountId, VMContext};
-    use near_sdk::Balance;
 
     use super::*;
 
     /// Used for asserts_eq.
     /// TODO: replace with derive when https://github.com/near/near-sdk-rs/issues/165
     impl Debug for MultiSigRequest {
         fn fmt(&self, _f: &mut Formatter<'_>) -> Result<(), Error> {
-            panic!("Should not trigger");
+            env::panic_str("Should not trigger");
```

Comment:
you don't need to update tests, as they are not contributing to the size of the contract.

Reply:
ah, yea, was thinking about that too but decided to go all in)